.github/workflows/release.yml

```diff
@@ -0,0 +1,25 @@
+name: Release
+on:
+  push:
+    branches: [ master ]
+  release:
+    types: [ published ]
+jobs:
+  release:
+    runs-on: ubuntu-20.04
+    steps:
+      - uses: actions/checkout@v2.4.0
+        with:
+          fetch-depth: 0
+      - uses: olafurpg/setup-scala@v13
+        with:
+          java-version: adopt@1.8
+      - name: Coursier cache
+        uses: coursier/cache-action@v6
+      - name: sbt ci-release ${{ github.ref }}
+        run: ./sbt ci-release
```

Comment:
That didn’t work :-/
Should be just `sbt ci-release`

See https://github.com/scalacenter/sbt-missinglink/runs/4314882199?check_suite_focus=true

Reply:
Yes, sorry
https://github.com/scalacenter/sbt-missinglink/pull/44